std_msgs/Header header
geometry_msgs/Pose[] poses
uint16 controlId